<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProgConfigSuitsDisplayPage</class>
 <widget class="QWizardPage" name="CProgConfigSuitsDisplayPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/newPrefix/resources/BridgeSceneGG7.bmp</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>You can choose the order in which card suits are displayed onscreen.  The logically natural ordering is Spades - Hearts - Diamonds - Clubs, but this would put the two red suits next to each other.  Hence, the default in ZBridge is S-H-C-D.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shcdSuitSequence">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/resources/Suit_Order_1.bmp</normaloff>:/newPrefix/resources/Suit_Order_1.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="shdcSuitSequence">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/resources/Suit_Order_2.bmp</normaloff>:/newPrefix/resources/Suit_Order_2.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dchsSuitSequence">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/resources/Suit_Order_3.bmp</normaloff>:/newPrefix/resources/Suit_Order_3.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cdhsSuitSequence">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/resources/Suit_Order_4.bmp</normaloff>:/newPrefix/resources/Suit_Order_4.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="hsdcSuitSequence">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/resources/Suit_Order_5.bmp</normaloff>:/newPrefix/resources/Suit_Order_5.bmp</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>310</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Some people also prefer to have the dummy hand's trumps suit on the left.  Click below if you wish to enable this option.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="indent">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showDummyTrumpsOnLeft">
         <property name="text">
          <string>Yes, always show dummy's trumps on the left.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
